powerauth-push-server/src/main/resources/application.properties

```diff
@@ -44,6 +44,9 @@ powerauth.push.service.applicationEnvironment=
 # PowerAuth 2.0 Push Campaign Setup
 powerauth.push.service.campaign.batchSize=100000
 
+# Whether persistent storing of sent messages is enabled
+powerauth.push.service.message.storage.enabled=true
```

Comment:
Let's set this to false by default - I believe that storing the push messages proved useless so far and only seem to hurt performance. For migrating customers, we need to explicitly mention this behaviour change...

Reply:
Ok, see: https://github.com/wultra/powerauth-push-server/wiki/PowerAuth-Push-Server-0.21.0#storing-of-sent-push-messages-disabled-by-default